docs/src/Experimental/divisors.md

```diff
@@ -0,0 +1,71 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["divisors.md"]
+```
+
+# Divisors
+
+In order to consider divisors on curves, we restrict our attention to smooth and irreducible curves.
+
+Let ``C`` be an affine or projective plane curve defined by an irreducible equation ``F``. Then any polynomial function ``G`` which is not divisible by ``F`` will vanish on ``C`` only at finitely many points. A way to encode these points together with their intersection multiplicities is to consider a divisor. A divisor on a curve is a formal finite sum of points of the curve with integer coefficients. A natural operation of addition can be defined on the set of divisors of a curve, which turns it into an Abelian group.
+
+
+## Constructors
+
+Divisors on curves are here introduced as a dictionary associating a point on the curve to its multiplicity.
+
+```@docs
+AffineCurveDivisor
+```
+
+```@docs
+ProjCurveDivisor
+```
+
+To define the divisor ``0`` of the group of divisors, one uses the following method:
```

Comment:
Here and elsewhere: please use (here and everywhere else) single backticks for inline code like this (and three for code blocks), but not double backticks. I.e.,:
```suggestion
To define the divisor `0` of the group of divisors, one uses the following method:
```